src/test/java/com/commercetools/project/sync/CliRunnerTest.java

```diff
@@ -469,6 +499,8 @@ void run_WithSyncAsArgumentWithAllArgWithRunnerName_ShouldExecuteAllSyncers() {
         .thenReturn(CompletableFuture.completedFuture(PagedQueryResult.empty()));
     when(sourceClient.execute(any(CartDiscountQuery.class)))
         .thenReturn(CompletableFuture.completedFuture(PagedQueryResult.empty()));
+    when(sourceClient.execute(any(StateQuery.class)))
+        .thenReturn(CompletableFuture.completedFuture(PagedQueryResult.empty()));
```

Comment:
same comment as above. move this lines a little bit to be more consistent. Type->Product->State-> ...

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/133#discussion_r466989525

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/133#discussion_r467259183

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/133/files/cfd976b1c0e5a91c50936a2601d6fdc5cea1f60a#r467711109